timers.js

```diff
@@ -1,4 +1,34 @@
+var secondsShown = 0;
+var clock;
+var timeStarted = true;
 
 
+$("#start").on("click", startWatch); // Start
 
+function startWatch(){
+    if (timeStarted == true){ // added this if statement to prevent multiple intervals but wont start again after Pause
+     clock = setInterval(function(){
+      $("#timer").text(secondsShown); secondsShown++;
+    }, 1000);
+    timeStarted = false;
+}
```

Comment:
😬 Indent properly! You should never have two curlies vertically right on top of each other.